.changeset/weak-yaks-learn.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-azure-devops': patch
+'@backstage/plugin-azure-devops-backend': patch
+'@backstage/plugin-azure-devops-common': patch
+---
+
+Added README card for Azure Devops
```

Comment:
As someone who reads the release notes to help upgrading having some more details here are really helpful, maybe at least a link to the new section in the README?